.travis.yml

```diff
@@ -27,11 +32,9 @@ before_install:
 
 install:
   - python setup.py -q develop
-  - if [[ $TRAVIS_PYTHON_VERSION == "2.7" ]]; then pip install git+git://github.com/tinio/pysqlite.git@extension-enabled#egg=pysqlite; fi
 
 script:
   - coverage run --source=openwisp_firmware_upgrader runtests.py
```

Comment:
Please add the standard `openwisp-utils` qa checks as well! :smile:

Reply:
(check the other repositories's travis.yml for it), example : 
```
openwisp-utils-qa-checks \
    --migration-path \
      "./openwisp_controller/config/migrations \
      ./openwisp_controller/connection/migrations \
      ./openwisp_controller/geo/migrations \
      ./openwisp_controller/pki/migrations" \
    --migrations-to-ignore "12 0 0 4" \
    --migration-module "config connection geo pki";
```
from controller: https://github.com/openwisp/openwisp-controller/blob/master/.travis.yml

Reply:
Hello ! @atb00ker I saw separate  [issue](https://github.com/openwisp/openwisp-firmware-upgrader/issues/1) on it :smile: can I work on it after finishing my current one ?

Reply:
Let's do the improvement of the build (qa-checks and so on) in the other issue